### PR TITLE
Adding wisenet cameras and move to use httpx

### PR DESCRIPTION
**Pull request by Charlie-Henry**
https://github.com/cityofaustin/atd-data-tech/issues/9594

Dropped aiohttp and swapped it with httpx because it supports Digest authentication which is needed for this new camera manufacturer. There are about 200 wisenet cameras deployed already so this will greatly expand our [screenshots](https://data.mobility.austin.gov/traffic-cameras)!

We will need to remember to add our new environment variables at deployment.
- `WISENET_USERNAME`
- `WISENET_PASSWORD`

### CODE REVIEW

File: cctv/process_images.py [hunk 1]
```diff
@@ -67,16 +65,18 @@ def create_camera(record, fallback_img):
     Raises:
         ValueError: raised by <Camera > if not ip, id, or model
     """
-    ip = record.get(IP_FIELD)
+    ip = record.get(IP_FIELD).strip()
```
Comment (Charlie-Henry) [inline cctv/process_images.py:68]: noticed some IPs in knack had trailing/leading spaces so this will clean these up

File: cctv/camera.py [hunk 2]
```diff
@@ -60,6 +62,13 @@ def __init__(
         self.fallback_img = fallback_img
         self.exception_limit = exception_limit
 
+        # Digest auth for Hanwha AKA Wisenet cameras
+        if self.model.lower() == "wisenet":
+            self.auth = httpx.DigestAuth(WISENET_USERNAME, WISENET_PASSWORD)
```
Comment (johnclary) [inline cctv/camera.py:67]: 🎉

File: cctv/camera.py [hunk 3]
```diff
@@ -124,45 +135,46 @@ async def download(self, session: aiohttp.ClientSession) -> bool:
...
-            return await response.content.read()
+
+        try:
+            response = await session.get(self.url, auth=self.auth)
+            response.raise_for_status()
+        except httpx.RequestError as exc:
+            return self._raise_exception(
+                f"Request error to {exc.request.url}({exc.__class__})",
+            )
+        except httpx.HTTPStatusError as exc:
```
Comment (johnclary) [inline cctv/camera.py:158]: very nice that you got these new exception types ironed out 🙌

File: readme.md [hunk 4]
```diff
@@ -14,13 +14,13 @@ This module fetches thumbnail images from the traffic cameras network and upload
 
 - The [public traffic camera dataset](https://data.austintexas.gov/Transportation-and-Mobility/Traffic-Cameras/b4k4-adkb) provides a listing of all available cameras
 
-- The [traffic cameras dashboards(https://data.mobiltiy.austin.gov) is a map for searching/viewing cameras
+- The [traffic cameras dashboards](https://data.mobiltiy.austin.gov) is a map for searching/viewing cameras
```
Comment (johnclary) [inline readme.md]: maybe update this to the full path - `https://data.mobility.austin.gov/traffic-cameras` ?

File: requirements.txt [hunk 5]
```diff
@@ -1,3 +1,4 @@
 aiohttp==3.7.*
```
Comment (johnclary) [inline requirements.txt]: can we remove this?

**Review by johnclary — APPROVED** [review]
I know this is very hard to test. You did send me a link to [this wisenet image](https://cctv.austinmobility.io/image/1.jpg) so i'm cool taking your word for it.

This is awesome work, @Charlie-Henry! so awesome how you were able to patch in `httpx` seamlessly. This is a major improvement for our camera images! 🎉 🚢 🚢

**Pull request merged**